src/queue.js

```diff
@@ -212,4 +215,16 @@ Queue.prototype._dequeuePendingRequest = function(item) {
   }
 };
 
+Queue.prototype._maybeLog = function(item) {
+  if (this.logger && this.options.verbose) {
+    var message = _.get(item, 'data.body.trace.exception.message');
```

Comment:
As far as I can tell, https://github.com/rollbar/rollbar.js/blob/7717f62eecedb3f5e7400b633ab8070ed2e45b6b/src/notifier.js#L577 is the only use of the verbose option in the old rollbar.js this basically matches that. I am not sure exactly what we want to log in verbose mode. There were also a lot of info logging to the console going on before which we have essentially removed. I have to dig in to see how that was configured, but it did not use the verbose option.